test/std-test.cc

```diff
@@ -16,7 +16,11 @@ TEST(std_test, path) {
             "\"foo\\\"bar.txt\"");
 
 #  ifdef _WIN32
-  EXPECT_EQ(fmt::format("{}", std::filesystem::path(L"Файл.txt")),
+  // File.txt in Russian.
+  // Do not use utf-8 encoded chars in this file.
+  const wchar_t cyrillic_path[] = {0x424, 0x430, 0x439, 0x43b, 0x2e,
+                                   0x74,  0x78,  0x74,  0};
+  EXPECT_EQ(fmt::format("{}", std::filesystem::path(cyrillic_path)),
             "\"\xd0\xa4\xd0\xb0\xd0\xb9\xd0\xbb.txt\"");
```

Comment:
I think you can use UTF-8 without escape sequences here as we do in `format-test`: https://github.com/fmtlib/fmt/blob/2471875867fbf6febdf119f287296a93843ae66b/test/format-test.cc#L175